config/core/build-configs-cip.yaml

```diff
@@ -0,0 +1,125 @@
+trees:
+  cip:
+    url: "https://git.kernel.org/pub/scm/linux/kernel/git/cip/linux-cip.git"
+
+# Default config with full build coverage
+build_configs_defaults:
+  variants:
+    gcc-10: &default_gcc-10
+      build_environment: gcc-10
+
+      fragments: &default_fragments
+        - 'debug'
+        - 'kselftest'
+        - 'tinyconfig'
+
+      architectures: &default_architectures
+
+        arm: &arm_arch
+          base_defconfig: 'multi_v7_defconfig'
+          extra_configs:
+            - 'allmodconfig'
+            - 'allnoconfig'
+            - 'multi_v7_defconfig+CONFIG_CPU_BIG_ENDIAN=y'
+            - 'multi_v7_defconfig+CONFIG_SMP=n'
+            - 'multi_v7_defconfig+CONFIG_EFI=y+CONFIG_ARM_LPAE=y'
+            - 'multi_v7_defconfig+CONFIG_THUMB2_KERNEL=y'
+          fragments: [crypto, ima]
+
+        arm64: &arm64_arch
+          extra_configs:
+            - 'allmodconfig'
+            - 'allnoconfig'
+            - 'defconfig+CONFIG_CPU_BIG_ENDIAN=y'
+            - 'defconfig+CONFIG_RANDOMIZE_BASE=y'
+          fragments: [crypto, ima]
+
+        i386: &i386_arch
+          base_defconfig: 'i386_defconfig'
+          extra_configs: ['allnoconfig']
+
+        x86_64: &x86_64_arch
+          base_defconfig: 'x86_64_defconfig'
+          extra_configs:
+            - 'allmodconfig'
+            - 'allnoconfig'
+            - 'x86_64_defconfig+x86-chromebook+kselftest'
+          fragments: [crypto, ima, x86_kvm_guest, x86-chromebook]
+
+  reference:
+    tree: mainline
+    branch: 'master'
+
+build_configs:
+  cip_4.4:
+    tree: cip
+    branch: 'linux-4.4.y-cip'
+    variants:
+      gcc-10:
+        build_environment: gcc-10
+        architectures:
+          arm: *arm_arch
+          i386: *i386_arch
+          x86_64: *x86_64_arch
+
+  cip_4.4-rt:
+    tree: cip
+    branch: 'linux-4.4.y-cip-rt'
+    variants:
+      gcc-10:
+        build_environment: gcc-10
+        architectures:
+          arm: *arm_arch
+          i386: *i386_arch
+          x86_64: *x86_64_arch
+
+  cip_4.19:
+    tree: cip
+    branch: 'linux-4.19.y-cip'
+    variants:
+      gcc-10:
+        build_environment: gcc-10
+        architectures:
+          arm:
+            <<: *arm_arch
+            extra_configs:
+              - 'cip://4.19.y-cip/arm/qemu_arm_defconfig'
+          arm64:
+            <<: *arm64_arch
+            extra_configs:
+              - 'cip://4.19.y-cip/arm64/qemu_arm64_defconfig'
+          x86_64:
+            <<: *x86_64_arch
+            extra_configs:
+              - 'cip://4.19.y-cip/x86/cip_qemu_defconfig'
+
+  cip_4.19-rt:
+    tree: cip
+    branch: 'linux-4.19.y-cip-rt'
+    variants:
+      gcc-10:
+        build_environment: gcc-10
+        architectures:
+          arm: *arm_arch
+          arm64: *arm64_arch
+          x86_64: *x86_64_arch
+
+  cip_5.10:
+    tree: cip
+    branch: 'linux-5.10.y-cip'
+    variants:
+      gcc-10:
+        build_environment: gcc-10
```

Comment:
The `kselftest` fragment can also be added at the top level, so it will apply to all architectures.

Reply:
followed stable